src-browserify/core/log.js

```diff
@@ -1,84 +1,54 @@
-/**
- * logging
- */
-
-(function() {
-
-    // error management
-    cdb.core.Error = Backbone.Model.extend({
-        url: cdb.config.REPORT_ERROR_URL,
-        initialize: function() {
-            this.set({browser: JSON.stringify($.browser) });
-        }
-    });
-
-    cdb.core.ErrorList = Backbone.Collection.extend({
-        model: cdb.core.Error,
-        enableTrack: function() {
-          var old_onerror = window.onerror;
-          window.onerror = function(msg, url, line) {
-              cdb.errors.create({
-                  msg: msg,
-                  url: url,
-                  line: line
-              });
-              if (old_onerror)
-                old_onerror.apply(window, arguments);
-          };
-        }
-    });
-
-    /** contains all error for the application */
-    cdb.errors = new cdb.core.ErrorList();
-
-
-    // error tracking!
-    if(cdb.config.ERROR_TRACK_ENABLED) {
-      cdb.errors.enableTrack();
```

Comment:
`cdb.config.ERROR_TRACK_ENABLED` is always set to false, so this code path is unreachable, so :fire: it